src/test/strtol.cc

```diff
@@ -234,21 +234,6 @@ TEST(StrictSICast, Error) {
     (void)strict_si_cast<int>("1T", &err);
     ASSERT_NE(err, "");
   }
-  {
```

Comment:
i think it's good to have more coverage for `strict_si_cast<int64_t>` even if `int64_t` is identical to `long long int` on some platforms.